modules/commons/src/main/java/org/apache/synapse/commons/resolvers/ResolverFactory.java

```diff
@@ -31,8 +31,9 @@
     private static final int RESOLVER_INDEX = 2;
     private static ResolverFactory resolverFactory = new ResolverFactory();
     private final Pattern rePattern = Pattern.compile("(\\$)([a-zA-Z0-9]+):([_a-zA-Z0-9]+)");
-    private static final Log log = LogFactory.getLog(ResolverFactory.class);
+    private static final Log LOG = LogFactory.getLog(ResolverFactory.class);
     private static final String SYSTEM_VARIABLE_PREFIX = "$SYSTEM";
+    private static final String FILE_PROPERTY_VARIABLE_PREFIX = "$FILEPROPERTY";
```

Comment:
Would $FILE be better?

Reply:
Fix with the commit https://github.com/wso2/wso2-synapse/pull/1431/commits/1a1c4472a75a7c34b4dc887a3d4dd499d3fbc5f7